charts/agent-control/charts/agent-control-deployment/templates/rbac.yaml

```diff
@@ -30,19 +30,6 @@ rules:
       - get
       - list
       - watch
-  - apiGroups: [ "apps" ]
-    resources:
-      - deployments/scale
-    verbs:
-      - get
```

Comment:
isn't it used by the health checkers?

Reply:
a few lines above there is
```
  - apiGroups: [ "apps" ]
    resources:
      - deployments
      - daemonsets
      - statefulsets
    verbs:
      - get
      - list
      - watch
```

That is only to do a scale down of the ac POD when running the delete